File: 2.0/java/texts/getText.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.callstexts.model.Text;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        Text text = client.textsApi().get(379506003L, "fromNumber,toNumber,modified,finalTextResult");
    }
}

File: 2.0/csharp/texts/getText.cs
using CallfireApiClient;
using CallfireApiClient.Api.CallsTexts.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        // limit returned fields if needed
        Text text = client.TextsApi.Get(379506003, "fromNumber,toNumber,modified,finalTextResult");
    }
}
